src/PublicApiAnalyzers/Core/Analyzers/DeclarePublicApiAnalyzer.Impl.cs

```diff
@@ -599,7 +599,13 @@ private ApiName GetApiName(ISymbol symbol)
                 {
                     for (var current = symbol; current is not null; current = current.ContainingSymbol)
                     {
-                        foreach (var attribute in current.GetAttributes())
+                        var attributes = current.GetAttributes();
```

Comment:
💡 It's possible for both the property and the method to have attributes. For example:

```
[Experimental("A")]
public object? Value { [NotNull] get; }
```

Perhaps we extract the body of the `foreach` to a local function, and then call for both `current.GetAttributes()` and (if no experimental attribute was found) also call it for the containing property if the current method is an accessor.